File: notify/echoVars_1.js
// -----------------------------------------------------------------------------
console.log("+++ Start echo.");

const accountSid = process.env.ACCOUNT_SID;
const authToken = process.env.ACCOUNT_AUTH_TOKEN;
console.log("+ accountSid       :" + accountSid + ":");
console.log("+ authToken        :" + authToken + ":");

const notifyServiceSid = process.env.NOTIFY_SERVICE_SID;
console.log("+ notifyServiceSid :" + notifyServiceSid + ":");

const yourPhoneNumber = process.env.YOUR_PHONE_NUMBER;
const phoneNumber1 = process.env.PHONE_NUMBER_1;
const phoneNumber2 = process.env.PHONE_NUMBER_2;
const phoneNumber3 = process.env.PHONE_NUMBER_3;
console.log("+ yourPhoneNumber  :" + yourPhoneNumber + ":");
console.log("+ phoneNumber1     :" + phoneNumber1 + ":");
console.log("+ phoneNumber2     :" + phoneNumber2 + ":");
console.log("+ phoneNumber3     :" + phoneNumber3 + ":");

const twilio = require('twilio');
const client = twilio(accountSid,authToken);
const notify = client.notify.services(notifyServiceSid);
bindings = client.notify.services(notifyServiceSid).bindings.list({tag: "test"});
console.log("+ Helper library works.");

// -----------------------------------------------------------------------------
console.log("+++ Exit.");

File: notify/echoVars_1.py
print "+++ Echo environment variables."
import os
# ------------------------------------------------------------------------------
# Get your account SID and auth token from https://twilio.com/console.
account_sid = os.environ.get("ACCOUNT_SID")
auth_token = os.environ.get("AUTH_TOKEN")
print "+ ACCOUNT_SID =", account_sid
print "+ AUTH_TOKEN =", auth_token
#
accountPhoneNumber = os.environ.get("ACCOUNT_PHONE_NUMBER")
print "+ ACCOUNT_PHONE_NUMBER =", accountPhoneNumber
notifyServiceSid = os.environ.get("NOTIFY_SERVICE_SID")
print "+ NOTIFY_SERVICE_SID =", notifyServiceSid
print "+ The following are required when creating bindings:"
print "+ PHONE_NUMBER_1 =", os.environ.get("PHONE_NUMBER1")
print "+ PHONE_NUMBER_2 =", os.environ.get("PHONE_NUMBER2")
print "+ PHONE_NUMBER_3 =", os.environ.get("PHONE_NUMBER3")
print "+ PHONE_NUMBER_4 =", os.environ.get("PHONE_NUMBER4")
print ""
print "++ Test Twilio helper library."
from twilio.rest import Client
client = Client(account_sid, auth_token)
from datetime import date
bindings = client.notify.services(notifyServiceSid).bindings.list(start_date=date(2015, 8, 25))
print "+ Helper library works."
# ------------------------------------------------------------------------------
print ""
print "+++ Exit."

